src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import './Calculator.css';
+
+const Calculator = () => (
+  <section className="calculator">
+    <div className="screen">0</div>
+    <div className="work">
+      <div>AC</div>
+      <div>+/-</div>
+      <div>%</div>
+      <div className="function">/</div>
+      <div>7</div>
+      <div>8</div>
+      <div>9</div>
+      <div className="function">x</div>
+      <div>4</div>
+      <div>5</div>
+      <div>6</div>
+      <div className="function">-</div>
+      <div>1</div>
+      <div>2</div>
+      <div>3</div>
+      <div className="function">+</div>
+      <div className="zero">0</div>
+      <div>.</div>
+      <div className="function">=</div>
+    </div>
+  </section>
+);
```

Comment:
- [ ] As requested by the previous code reviewer, from the project requirement `Calculator.js` should have at least 1 child component. So please make sure that you create a child component for this Calculator component. :pray: